CMakeLists.txt

```diff
@@ -23,8 +23,10 @@ list(APPEND CMAKE_MODULE_PATH
 )
 
 # Set some useful global variables
-# this will hold herbert/horace_on.m and worker_v2.m
+# This will hold herbert/horace_on.m and worker_v2.m
 set(LOCAL_INIT_DIR "${CMAKE_BINARY_DIR}/local_init")
+# This sets the destination for mex build artifacts (used in PACE_AddMex)
+set(Horace_DLL_DIRECTORY "${CMAKE_SOURCE_DIR}/horace_core/DLL")
```

Comment:
is this still writing into Debug/Release while Herbert has been changed to use a common folder for both

Reply:
No, if you compare the top-level CMakeLists file in Herbert and this one, the `*_DLL_DIRECTORY` points to the same (relative) place. The `PACE_AddMex` function tells the compiler not to create Release/Debug folders within `*_DLL_DIRECTORY` and that file is shared between Herbert and Horace